src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -55,6 +55,7 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('name_converter')->defaultNull()->info('Specify a name converter to use.')->end()
                 ->scalarNode('path_segment_name_generator')->defaultValue('api_platform.path_segment_name_generator.underscore')->info('Specify a path name generator to use.')->end()
+                ->booleanNode('serialize_payload_constraint')->defaultFalse()->info('Enable the serialization of the payload field when a validation error is thrown.')->end()
```

Comment:
What do you think of something like:

```yaml
validator:
    serialize_payload: true
```

Some people are thinking about new options regarding the validator, this way we'll be feature-proof?

Reply:
Ok, I will change that tomorrow